tests/linen/linen_combinators_test.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for flax.linen.combinators."""
```

Comment:
These tests look really great!

Reply:
Thanks for taking a look! Updated to add test for the empty module.